paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -69,8 +69,11 @@ internal class FormViewModel @Inject internal constructor(
             lpmRepository.fromCode(formArguments.paymentMethodCode)
         ).formSpec.items
 
-        // Cards are a special case and already contain specs based on the configuration.
-        if (formArguments.paymentMethodCode != PaymentMethod.Type.Card.code) {
+        // Cards & Bacs debit are a special case and already contain specs based on the configuration.
+        if (
+            formArguments.paymentMethodCode != PaymentMethod.Type.Card.code &&
+            formArguments.paymentMethodCode != PaymentMethod.Type.BacsDebit.code
```

Comment:
`Bacs` uses `ContactInformationSpec` rather than individual fields for name & email. This isn't supported by the `PlaceholderField` type which denotes every value as a single element while `ContactInformationSpec` is an element with sub-elements.

I could make a change to `PlaceholderField` (only used internally) that allows for defining sub-elements.
```kotlin
// PlaceholderSpec.kt

//....
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
@Serializable
enum class PlaceholderField(val type: PlaceholderFieldType) {
    sealed interface PlaceholderFieldType {
        object Element : PlaceholderFieldType
        data class Container(val fields: List<PlaceholderField>) : PlaceholderFieldType
    }

    @SerialName("name")
    Name(PlaceholderFieldType.Element),

    @SerialName("email")
    Email(PlaceholderFieldType.Element),

    @SerialName("phone")
    Phone(PlaceholderFieldType.Element),

    @SerialName("contact_information")
    ContactInformation(PlaceholderFieldType.Container(listOf(Name, Email, Phone))),
    
    // ....
}
```